src/components/LibraryItem.tsx

```diff
@@ -184,9 +185,6 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
     }
 
     onLibraryItemClicked() {
-
-        // Toggle expansion state.
-        let currentlyExpanded = this.state.expanded;
-        this.setState({ expanded: !currentlyExpanded });
+        LibraryEntryPoint.dispatchEvent("libraryItemClick", this);
```

Comment:
We will get the `LibraryView` instance from here and then get it to raise the event instead of accessing the `Reactor` directly like this. Let me make a PR.